apps/emqx_conf/src/emqx_conf_schema.erl

```diff
@@ -325,8 +325,6 @@ fields("db") ->
            , default => mnesia
            , desc => """
 Select the backend for the embedded database.<br/>
-<strong>Important!</strong> This setting should be the same on all nodes in the cluster.<br/>
-<strong>Important!</strong> Changing this setting in the runtime is not allowed.<br/>
```

Comment:
Q: is this allowed now?  Won't it cause problems?

Reply:
With post commit hook it shouldn't cause much problems in EMQ X.